### ISSUE TITLE
Permissions viewer tree view starting closed?

### ISSUE THREAD

**Issue by wdhodges** [volkanceylan/Serenity#415]
Is there a way or a setting that we can enable that when you click on permissions, or role for a user the tree-view that is displayed started in the closed position versus open?

**Comment by volkanceylan** [comment]
```cs
        private void SetItems(List<PermissionCheckItem> items)
        {
            SlickTreeHelper.SetIndents(items,
                getId: x => x.Key,
                getParentId: x => x.ParentKey,
                setCollapsed: true);

            this.View.SetItems(items, true);
        }
```

**Comment by volkanceylan** [comment]
Do that in PermissionCheckEditor.cs

**Comment by wdhodges** [comment]
Thanks!!

**Comment by wdhodges** [comment]
This fix doesn't work in a fresh install of 2.1.4, when searching the solution for PermissionCheckEditor.cs it returns nothing.

**Comment by wdhodges** [comment]
Nevermind it - it's PermissionCheckEditor.ts
